integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/SlackSteps.java

```diff
@@ -0,0 +1,33 @@
+
+package com.redhat.service.bridge.integration.tests.steps;
+
+import java.time.Duration;
+
+import org.awaitility.Awaitility;
+
+import com.redhat.service.bridge.integration.tests.context.TestContext;
+import com.redhat.service.bridge.integration.tests.context.resolver.ContextResolver;
+import com.redhat.service.bridge.integration.tests.resources.SlackResource;
+
+import io.cucumber.java.en.Then;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SlackSteps {
+
+    private TestContext context;
+
+    public SlackSteps(TestContext context) {
+        this.context = context;
+    }
+
+    @Then("^Slack channel contains message with text \"([^\"]*)\" within (\\d+) (?:minute|minutes)$")
+    public void slackActionTest(String messageText, int timeoutMinutes) {
+        String messageTextWithoutPlaceholders = ContextResolver.resolveWithScenarioContext(context, messageText);
+        Awaitility.await()
+                .atMost(Duration.ofMinutes(timeoutMinutes))
```

Comment:
We should add the log in case of timeout:
```
.conditionEvaluationListener(new AwaitilityOnTimeOutHandler(() -> SlackResource.getListOfSlackMessages()))
```

wdyt ?

Reply:
Actually it is not needed in here as Awaitility is smart enough to remember last failing error:
```
  Then Slack channel contains message with text "hello world by ${bridge.mybridge.cloud-event.my-id.id}x" within 1 minute # com.redhat.service.bridge.integration.tests.steps.SlackSteps.slackActionTest(java.lang.String,int)
      org.awaitility.core.ConditionTimeoutException: Assertion condition defined as a lambda expression in com.redhat.service.bridge.integration.tests.steps.SlackSteps that uses java.lang.String 
Expecting actual:
  ["hello world by 522412bb-c9ca-4b1c-b86e-8a2c79559ec4",
    "hello world by 0c221181-3c29-47ee-8fef-abd40d7e3c89",
    "hello world by 378f7f12-9329-4bb2-bf51-0ad7346f95df",
...
to contain at least one of the following elements:
  ["hello world by 522412bb-c9ca-4b1c-b86e-8a2c79559ec4x"]
```

Reply:
true there is the containsAnyOf assertion